cmd/terramate/cli/cli.go

```diff
@@ -723,6 +724,10 @@ func (c *cli) createStack() {
 	})
 
 	if err != nil {
+		if c.parsedArgs.Create.IgnoreExisting &&
+			errors.IsKind(err, stack.ErrStackAlreadyExists) {
+			return
```

Comment:
maybe we should log something about this? so the user knows that the file was not copied?

Reply:
Makes sense, thanks :heart: